client-common/src/main/java/org/apache/cassandra/sidecar/common/ApiEndpointsV1.java

```diff
@@ -109,6 +109,8 @@ public final class ApiEndpointsV1
     public static final String ABORT_RESTORE_JOB_ROUTE = RESTORE_JOB_ROUTE + ABORT;
     public static final String RESTORE_JOB_PROGRESS_ROUTE = RESTORE_JOB_ROUTE + PROGRESS;
 
+    public static final String CLIENT_STATS_ROUTE = API_V1 + CASSANDRA + "/client-stats";
```

Comment:
I would like to have additional "context" in the URI between `cassandra` and the operation - `client-stats`. This was `storage` previously. Reviewers - Lmk your thoughts.